src/main/java/com/google/cloud/tools/ide/login/Account.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.ide.login;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.common.base.Preconditions;
+
+import javax.annotation.Nullable;
+import java.net.URL;
+
+/**
+ * Represents a single logged-in account.
+ */
+class Account {
```

Comment:
as discussed below, you should add equals and hashCode methods to this class such that two accounts are equals iff the emails are equal.